src/Standards/PSR12/ruleset.xml

```diff
@@ -105,7 +105,7 @@
     <!-- checked by PSR12.Namespaces.CompoundNamespaceDepth -->
 
     <!-- When wishing to declare strict types in files containing markup outside PHP opening and closing tags, the declaration MUST be on the first line of the file and include an opening PHP tag, the strict types declaration and closing tag. -->
-    <!-- Declare statements MUST contain no spaces and MUST be exactly declare(strict_types=1) (with an optional semi-colon terminator). -->
+    <!-- Declare statements MUST contain no spaces and MUST be exactly declare(strict_types=1) (with an optional semicolon terminator). -->
```

Comment:
This is a literal quote from PSR-12: https://www.php-fig.org/psr/psr-12/#3-declare-statements-namespace-and-import-statements (towards the end of the section).

To change this one, I believe a PR to PSR-12 would be needed first ?

Note: this does appear to have been changed for PSR-PER: https://www.php-fig.org/per/coding-style/#3-declare-statements-namespace-and-import-statements

Reply:
PR for PSR-12 has been created, let's see how that goes.

Reply:
@DannyvdSluijs Thanks. If you remove this one change from this PR, we can get it merged. Once the upstream PR has been merged (if accepted), I'd be happy to merge the change to the ruleset too (in a follow-up PR). Does that work for you ?

Reply:
Yes fine with me. I've reverted this file to its original and rebased on master. Let me know if you need anything else.

Reply:
Well actually that went pretty fast in the PHP-FIG repo, see https://github.com/php-fig/fig-standards/pull/1322

I would guess it would be okay to correct it here then as well?

Reply:
Indeed 👍🏻

Reply:
Dropped that last commit. Everything should be okay now and the branch is up to date.